### PR TITLE
don't include removal diff in copy-paste code, fix tests

### PR DESCRIPTION
**Pull request by jenweber**
This PR makes it so that the removal diff isn't copy-pasted.

Before (all text can be highlighted for copy-paste):
Screen Shot 2019-11-06 at 11 09 34 AM

After (red diff text is excluded from copy-paste):
Screen Shot 2019-11-06 at 11 09 06 AM

See https://github.com/ember-learn/super-rentals-tutorial/issues/106 for motivation.

### CODE REVIEW

**Comment by chancancode** [comment]
This is great! Related: I would love to make the diff toggle-able also (Before/After/Both). Most of the time it can probably default to After by default, but having the diff makes it easier to spot the changes, especially it’s just a small change in a long existing file.

File: addon/styles/_prism-overrides.scss [hunk 1]
```diff
@@ -32,6 +32,11 @@ code .diff-operator {
   user-select: none;
 }
 
+code .diff-deletion {
+  // don't include removed code in copy-paste
+  user-select: none;
+}
```
Comment (chancancode) [inline addon/styles/_prism-overrides.scss]: Actually I’m not sure this does what you want. I think at least in some browsers, this only affects how the selection looks visually, but doesn’t prevent them from being copied if you select the surrounding text like you did in the screenshot. Have you tested it on multiple browsers?

Comment (jenweber) [reply addon/styles/_prism-overrides.scss]: I have not, but I just looked it up on caniuse and it seems I need to prefix it. Thanks for bringing this up!

**Comment by jenweber** [comment]
I think making the diffs toggleable would be very doable! That work would be done in this repo as well. I'm thinking a basic if statement on the code container classes and a buttons component would do it.

**Comment by MelSumner** [comment]
@jenweber do you think that this PR is an MVP and we could file a new issue and iterate?

**Comment by jenweber** [comment]
Tests were failing with a warning to not use the `{{array}}` helper so I changed it to a computed property.

**Comment by jenweber** [comment]
Some work was done on another branch to fix the test failures: https://github.com/ember-learn/guidemaker-ember-template/pull/38

Here, I updated versions of ember-composable-helpers. 2.x had an `array` helper included.

**Comment by jenweber** [comment]
@mansona @MelSumner tests are passing now after I updated the composable-helpers version.

**Pull request merged**